frontend/packages/dev-console/integration-tests/features/web-terminal/web-terminal-devuser.feature

```diff
@@ -2,26 +2,30 @@
 Feature: Web Terminal for Developer user
               As a developer user, I should be able to use web terminal
 
-
         Background:
             Given user has logged in as basic user
+              # for correct checking of the user story we can use just an one project with DevWorkspace operator
```

Comment:
I don't think we should reference DevWorkspace Operator here.
+ typically basic user does not have rights in DWO namespace.